File: content/atlas/source/includes/fts/search-index-management/csharp/ListIndexes.cs
using MongoDB.Bson;
using MongoDB.Driver;

// connect to your Atlas deployment
var uri = "<connection-string>";

var client = new MongoClient(uri);

var db = client.GetDatabase("<databaseName>");
var collection = db.GetCollection<BsonDocument>("<collectionName>");

// list your MongoDB Search indexes
var result = collection.SearchIndexes.List().ToList();
foreach (var index in result)
{
    Console.WriteLine(index);
}

File: content/search/source/includes/index/manage-indexes/code-snippets/view-index/nodejs/list-indexes.js
import { MongoClient } from "mongodb";

// connect to your Atlas deployment
const uri =  "<connection-string>";

const client = new MongoClient(uri);

async function run() {
  try {
    const database = client.db("<databaseName>");
    const collection = database.collection("<collectionName>");

    // run the helper method
    const result = await collection.listSearchIndexes("<index-name>").toArray();
    console.log(result);
  } finally {
    await client.close();
  }
}
run().catch(console.dir);
